lesson-4/README.md

```diff
@@ -0,0 +1,217 @@
+# Prometheus - Kubernetes monitored
+
+## Table of Contents
+- [Prerequisites](#Prerequisites)
+- [Run Prometheus in Kubernetes](#run-prometheus-in-kubernetes)
+  - [Prepare the Namespace](#prepare-the-namespace)
+  - [Deploying Prometheus](deploying-prometheus)
+    - [ConfigMap](#configmap)
+    - [Prometheus](#prometheus)
+    - [Prometheus Service](#prometheus-service)
+- [Run Grafana]()
+- [Prometheus Node Exporter]()
+
+## Prerequisites
+- `git clone https://github.com/prgcont/workshop-k8s && cd workshop-k8s/src/prometheus`
+- [Install Hypervisor](https://github.com/prgcont/workshop-k8s/tree/master/lesson-1#install-hypervisor)
+- [Install Minikube](https://github.com/prgcont/workshop-k8s/tree/master/lesson-1#install-minikube)
+- [Install Kubeectl](https://github.com/prgcont/workshop-k8s/tree/master/lesson-1#install-minikube)
+- [Test Your Setup](https://github.com/prgcont/workshop-k8s/tree/master/lesson-1#run-minikube)
+
+## Run Prometheus in Kubernetes
+
+### Prepare the Namespace
+We're going to follow the best practices and run everything monitoring-related in a special namespaces, the namespaces will be named 'monitoring'. To create the new namespace in Kubernetes run:
+
+```bash
```

Comment:
Probably separate into two blocks, one for creating namespaces second for checking